packages/asc-scrapper/src/mappers.ts

```diff
@@ -1,83 +1,116 @@
+import {
+    TimetableClass,
+    TimetableCommonGroup,
+    TimetableDay,
+    TimetablePeriod,
+    TimetableRoom,
+    TimetableStudent,
+    TimetableSubject,
+    TimetableTeacher,
+    TimetableTimeSlot,
+    TimetableWeek,
+    slugify,
+} from '@timetable-api/common';
 import {
     ClassesTableRow,
     ClassroomsTableRow,
-    Clazz,
-    Division,
-    DivisionsTableRow,
-    Group,
+    DaysTableRow,
     GroupsTableRow,
     PeriodsTableRow,
-    Room,
-    Student,
     StudentsTableRow,
-    Subject,
     SubjectsTableRow,
-    Teacher,
     TeachersTableRow,
-    TimeSlot,
+    TermsTableRow,
+    WeeksTableRow,
 } from './types.js';
 import { parseTime } from './utils.js';
 
-export const mapPeriodsTableRow = (row: PeriodsTableRow): TimeSlot => ({
+const weekdayIsoNumber: Record<string, number> = {
+    poniedzialek: 1,
+    wtorek: 2,
+    sroda: 3,
+    czwartek: 4,
+    piatek: 5,
+    sobota: 6,
+    niedziela: 7,
+};
+
+export const mapPeriodsTableRow = (row: PeriodsTableRow): TimetableTimeSlot => ({
     id: row.id,
     name: row.name,
     short: row.short,
     beginMinute: parseTime(row.starttime),
     endMinute: parseTime(row.endtime),
 });
 
-export const mapClassroomsTableRow = (row: ClassroomsTableRow): Room => ({
+export const mapDaysTableRow = (row: DaysTableRow): TimetableDay => ({
+    id: row.id,
+    name: row.name,
+    short: row.short,
+    isoNumber: weekdayIsoNumber[slugify(row.name)],
```

Comment:
Zmień typ `weekdayIsoNumber` z `Record<string, number>` na `Partial<Record<string, number>>`.
`weekdayIsoNumber[slugify(row.name)]` dla niepoprawnej nazwy dnia tygodnia zwróci `undefined`, ale TS o tym nie wie, a typ `isoNumber` to `number | null`